syntax = "proto2";
option optimize_for = LITE_RUNTIME;
option java_package = "ProtoMessages";
option java_outer_classname = "MessageRequestRoomPlayerDuelistBase";

message MessageRequestRoomPlayerDuelist {
    required int64 roomId = 1 [default = 0];
}